spec/index.html

```diff
@@ -187,10 +187,10 @@ <h3>Triple Terms</h3>
       it must also appear in a graph as an <a>asserted triple</a>.</p>
 
     <p>A <a>triple term</a> can be used as the object of a <a>triple</a> with the predicate <code>rdf:reifies</code>,
```

Comment:
If the `such a ...` change is retained, the line preceding it must end with a semicolon instead of a comma.
```suggestion
    <p>A <a>triple term</a> can be used as the object of a <a>triple</a> with the predicate <code>rdf:reifies</code>;
```